### PR TITLE
feat: support user for mobile isAppInstalled

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium-adb/pull/739

### CODE REVIEW

File: lib/commands/execute.js [hunk 1]
```diff
@@ -110,5 +110,5 @@ export async function executeMobile(mobileCommand, opts = {}) {
         `Only ${_.keys(mobileCommandsMapping)} commands are supported.`,
     );
   }
-  return await this[mobileCommandsMapping[mobileCommand]](opts);
+  return await this[/**@type {string}*/(mobileCommandsMapping[mobileCommand])](opts);
```
Comment (KazuCocoa) [inline lib/commands/execute.js:113]: this is to not fail build

File: lib/commands/app-management.js [hunk 2]
```diff
@@ -17,13 +17,19 @@ export const APP_STATE = /** @type {const}  */ ({
...
+
 /**
  * @this {AndroidDriver}
  * @param {string} appId
+ * @param {IsAppInstalledOptions} [opts={}]
  * @returns {Promise<boolean>}
  */
-export async function isAppInstalled(appId) {
-  return await this.adb.isAppInstalled(appId);
+export async function isAppInstalled(appId, opts) {
```
Comment (mykola-mokhnach) [inline lib/commands/app-management.js:31]: should opts have a default value?

Comment (KazuCocoa) [reply lib/commands/app-management.js]: yea, 2cfe880b23eaea0317c3d70af05eac96de32b812

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
isAppInstaled -> isAppInstalled

**Pull request merged**